modules/vaos/spec/request/v1/healthcare_service_request_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'VAOS::V1::HeathcareService', type: :request do
+  include SchemaMatchers
+
+  before do
+    Flipper.enable('va_online_scheduling')
+    sign_in_as(user)
+    allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
+  end
+
+  describe '/vaos/v1/HealthcareService' do
+    context 'with a loa1 user' do
+      let(:user) { FactoryBot.create(:user, :loa1) }
+
+      it 'returns a forbidden error' do
+        get '/vaos/v1/HealthcareService'
+        expect(response).to have_http_status(:forbidden)
+        error_object = JSON.parse(response.body)
+        expect(error_object['resourceType']).to eq('Healthcare_service')
+        expect(error_object['issue'].first['details']['text'])
+          .to eq('You do not have access to online scheduling')
+      end
+    end
+
+    context 'with a loa3 user' do
+      let(:user) { build(:user, :vaos) }
+
+      context 'FHIR HealthcareService Resource search' do
+        context 'returns a valid response with many results' do
+          let(:expected_body) do
+            YAML.load_file(
+              Rails.root.join(
+                'spec', 'support', 'vcr_cassettes', 'vaos', 'fhir', 'healthcare_service', 'search_200.yml'
+              )
+            )['http_interactions'].first.dig('response', 'body', 'string')
+          end
+
+          let(:query_string) { '?organization.identifier=983&_include=HealthcareService:location' }
+
+          it 'returns a 200' do
```

Comment:
you could `xit` specs to skip them and get CI to pass if you're going to record VCR cassettes on staging

Reply:
Thanks, nice tip will keep that in mind.